pages/draft-06/migration-notes.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Draft 04 to Draft 06
+section: docs
+---
+
+
+### Introduction
+
+Draft-06 of the JSON Schema specification introduced a number of changes over Draft-04 that improve keyword clarity, validation flexibility, and alignment with common development practices. Migration from Draft-04 to Draft-06 involves both structural and behavioral adjustments to keywords, formats, and validation logic. This guide covers the key differences and provides practical steps for implementers updating their schemas and tooling.
+
+### Keyword Changelog
+
+| Keyword (Draft-04)                               | Keyword (Draft-06)                              | Specification | Keyword Type       | Behavior Details                                                                            |
+| ------------------------------------------------ | ----------------------------------------------- | ------------- | ------------------ | ------------------------------------------------------------------------------------------- |
+| `id`                                             | `$id`                                           | Core          | Identifier         | Replaces `id` with `$id` to align with JSON-LD and emphasize its identifier role.           |
+| `exclusiveMinimum`, `exclusiveMaximum` (boolean) | `exclusiveMinimum`, `exclusiveMaximum` (number) | Validation    | Numeric Boundaries | Previously booleans tied to `minimum`/`maximum`; now standalone numeric values.             |
+| `definitions`                                    | `definitions`                                   | Core          | Schema Reuse       | Still available but less emphasized in later drafts in favor of `$defs` (introduced later). |
```

Comment:
I'm not sure about this one. `$defs` was introduced in 2019-09. Not sure we want to mention it at all here